Bio/SeqFeature.py

```diff
@@ -851,6 +851,16 @@ def __iter__(self):
             for i in range(self._start, self._end):
                 yield i
 
+    def __eq__(self, other):
+        """Check if start, end and strand of this location are the same as in the other location"""
+        if not isinstance(other, FeatureLocation):
+            return False
+        return self._start == other.start and self._end == other.end and self._strand == other.strand
```

Comment:
I actually didn't look at `FeatureLocation` as I initially started to compare `CompoundLocation`s, where both `ref` and `ref_db` are hardcoded to `None`. Didn't occur to me to look at `FeatureLocation`.